### PR TITLE
docs: add README files for Rust and Node.js

### PR DESCRIPTION
**Pull request by Dolu89**
Add some documentation in README.md (for Rust and Node.js)
#20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```sh
+cargo run --package una-proto-builder
+cargo build --bin una-cli
+```
+
+### Choose your backend
+#### LND
+
+```sh
+una-cli -- --backend LndRest --url https://127.0.0.1:8081 --macaroon HEX_MACAROON --tls_certificate HEX_CERTIFICATE
```
Comment (louneskmt) [inline README.md]:
```suggestion
una-cli --backend LndRest --url https://127.0.0.1:8081 --macaroon HEX_MACAROON --tls_certificate HEX_CERTIFICATE
```
I don't think we need those if we execute the binary and not `cargo run`

File: bindings/una-js/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Python bindings for UNA
```
Comment (louneskmt) [inline bindings/una-js/README.md]:
```suggestion
# Node bindings for UNA
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# UNA - Universal Node API
+
+Una is a Lightning network node wrapper for multiple backends and for multiple programming languages
```
Comment (louneskmt) [inline README.md]:
```suggestion
Una is a Lightning network node wrapper for multiple backends and for multiple programming languages.
```

**Review by louneskmt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Dolu89) [reply README.md]: I tried, yes it's useless using the binary

**Comment by louneskmt** [comment]
We should be a bit for explicit about Una on the bindings README, don't you think? These will be the README of each package, they are not just subdirs README.

**Comment by louneskmt** [comment]
Looks good for now 👌

We will try in the future to have a docs website like the old LND REST API one, where you could see examples in several languages.

**Comment by louneskmt** [comment]
Should we merge this yet @Dolu89?

**Comment by Dolu89** [comment]
> Should we merge this yet @Dolu89?

Yes

**Review by louneskmt — APPROVED** [review]

**Comment by louneskmt** [comment]
Merged, thank you!

**Pull request merged**